container/config.yml

```diff
@@ -2,7 +2,7 @@ name: "NeoFS Container"
 safemethods: ["alias", "count", "containersOf", "get", "owner", "list", "eACL", "getContainerSize", "listContainerSizes", "iterateContainerSizes", "iterateAllContainerSizes", "version"]
 permissions:
   - methods: ["update", "addKey", "transferX",
-              "register", "registerTLD", "addRecord", "deleteRecords"]
+              "register", "registerTLD", "addRecord", "deleteRecords", "registerNewEpochRecipient"]
```

Comment:
It may sound a little NeoGo-ish, but how about `subscribeForNewEpoch`? A lot of other `register` methods exist in NeoFS contracts.

Reply:
goodish